src/main/java/frc/robot/commands/translationCommand.java

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class translationCommand extends CommandBase {
+  ProfiledPIDController pidControlX;
+  double currentX;
+  double desiredX;
+  double deltaX;
+
+
+  ProfiledPIDController pidControlY;
+  double currentY;
+  double desiredY;
+  double deltaY;
+
+  
+  /** Creates a new translation. */
+  public translationCommand( double deltaX, double deltaY) {
+    pidControlX = new ProfiledPIDController(0.1, 0.0, 0.0, 
+                                           new TrapezoidProfile.Constraints(3, 2));
+    pidControlY = new ProfiledPIDController(0.1, 0.0, 0.0, 
+                                           new TrapezoidProfile.Constraints(3,2));
```

Comment:
3 m/s and 2 m/s/s is very fast. It's VERY fast. To climb onto the charge station, you probably want 1 m/s and 1 m/s/s.

I'm planning to never let PathPlanner go above 2.5 m/s even though it might be able to reach 3 m/s.